benchmarks/web-search/client/Dockerfile

```diff
@@ -1,17 +1,12 @@
-FROM cloudsuite/java:openjdk7
+FROM clousuite/faban
 LABEL maintainer="Siddharth Gupta <siddharth.gupta@epfl.ch>"
 
 ENV FABAN_USER faban
 
-RUN apt-get update -y \
-	&& apt-get install -y --no-install-recommends telnet wget tar curl \
-	&& rm -rf /var/lib/apt/lists/* \
-	&& groupadd -r $FABAN_USER  \
-	&& useradd -r -g $FABAN_USER $FABAN_USER
 
-ENV BASE_PATH /usr/src
-ENV FABAN_HOME $BASE_PATH/faban
-ENV FABAN_VERSION 1.3
+
+
+
 ENV PACKAGES_URL http://lsi-www.epfl.ch/parsa
 ENV SEARCH_DRIVER_URL $PACKAGES_URL/search.tar.gz
 ENV FABAN_PORT 9980
```

Comment:
this should go to the faban image.

Reply:
These parameters are used while running faban and not while installing it. Shouldn't they be in the client itself? This would be useful if we want to use the same faban image while running other benchmarks.